app/javascript/data/models/schedules.js

```diff
@@ -0,0 +1,11 @@
+import { string, object } from 'yup';
+
+export const schedulesSchema = object().shape({
+  href: string().required(),
+  id: string().required(),
+  run_at: object()
```

Comment:
So the full object looks kinda like:

{
href:'foo'
schedules:[
{run_at:{
start_time: 'foo'
}}]}

Reply:
i am seeing:
```
      "schedules": [
        {
          "href": "http://0.0.0.0:8080/api/service_templates/5/schedules/31",
          "id": "31",
          "name": "Order ServiceTemplateTransformationPlan 5",
          "description": "Order ServiceTemplateTransformationPlan 5",
          "sched_action": {
            "args": [
              1,
              {
                
              },
              {
                "submit_workflow": true
              }
            ],
            "method": "queue_order"
          },
          "filter": null,
          "towhat": "ServiceTemplate",
          "run_at": {
            "interval": {
              "unit": "once"
            },
            "start_time": "2018-07-21T14:40:00Z",
            "tz": "UTC"
          },
```

Only attempting to get the fields the UI is using, but if you'd like to add more, fine w/ me ;)

Reply:
OH gotcha ok, yeah that's right, apologies, i thought this was reaching from an encapsulating service_template object 👍